src/requirements.ts

```diff
@@ -13,7 +13,8 @@ import * as cp from 'child_process';
 import * as path from 'path';
 import * as pathExists from 'path-exists';
 import * as expandHomeDir from 'expand-home-dir';
-import * as findJavaHome from 'find-java-home';
+import findJavaHome = require('find-java-home');
```

Comment:
```suggestion
import * as findJavaHome from 'find-java-home';
```
Why use require instead of import here?

Reply:
No idea, copied the same change they made in vscode-java. I though it might be important, but if you are confident it is harmless I'm ok to revert.

Reply:
I applied you fix